datafusion/expr/src/expr.rs

```diff
@@ -330,7 +331,7 @@ pub enum Expr {
     /// [`ExprFunctionExt`]: crate::expr_fn::ExprFunctionExt
     AggregateFunction(AggregateFunction),
     /// Call a window function with a set of arguments.
-    WindowFunction(WindowFunction),
+    WindowFunction(Box<WindowFunction>),
```

Comment:
I wonder, what is now 144 bytes?``

Reply:
Good question! I'm fairly new to Rust; what's a good way to analyze the enum variant sizes?

Reply:
It looks like `Alias` might be the culprit here (with some padding): 

`size_of::<Alias>()` is 136, followed by `size_of::<(DataType, Column)>()` (alias `OuterReferenceColumn`) with 128.

Reply:
Agree - although it seems the size of the changes will be lower than 272 to 144 (I hoped maybe there would be a single one to reduce it from 144 -> 80). 
Thanks for checking!